driver-core/src/test/java/com/datastax/driver/core/ControlConnectionTest.java

```diff
@@ -69,7 +69,7 @@ public long nextDelayMs() {
         };
 
         try {
-            ccm = CCMBridge.builder("test").withNodes(2).build();
```

Comment:
Some tests would need clusterProviders as well..will think about this some more.